### ISSUE TITLE
no ssd or nvme driver

### ISSUE THREAD

**Issue by arcanisgk** [memstechtips/UnattendedWinstall#136]
no ssd or nvme driver

**Comment by memstechtips** [comment]
Yes, that is not the XML file's doing though, that is the ISO provided by Microsoft. 
The image doesn't have those drivers and you'll have the same issue if you were trying to install Windows without the XML file.
